examples/ticks/tick-locators.py

```diff
@@ -73,3 +73,23 @@ def setup(ax, title):
 
 plt.tight_layout()
 plt.show()
+
+# %%
+#
+# .. admonition:: References
+#
+#    The following functions, methods, classes and modules are used in this example:
+#
+#    - `matplotlib.axis.Axis.set_major_locator`
+#    - `matplotlib.axis.Axis.set_minor_locator`
+#    - `matplotlib.axis.XAxis.set_ticks_position`
+#    - `matplotlib.axis.YAxis.set_ticks_position`
+#    - `matplotlib.ticker.NullLocator`
+#    - `matplotlib.ticker.MultipleLocator`
+#    - `matplotlib.ticker.FixedLocator`
+#    - `matplotlib.ticker.LinearLocator`
+#    - `matplotlib.ticker.IndexLocator`
+#    - `matplotlib.ticker.AutoLocator`
+#    - `matplotlib.ticker.MaxNLocator`
+#    - `matplotlib.ticker.LogLocator`
+# %%
```

Comment:
```suggestion
```

Pretty sure this line is causing the sphinx build to error

A quick survey of other similar references section this closing `%%` is not included